uwblueprint/sistema
# Template to send emails

# Notion ticket link


[Email Reminders](https://www.notion.so/uwblueprintexecs/Sistema-03b6ce1c4bc447b5945ab8b79cb00ea5?p=ae2ad9edf3b64073ae6dcc39a3fd3bc0&pm=s)



## Implementation description
- Use NodeMailer to be able to send emails
- Implemented unstyled template that allows us to send emails 



## Steps to test

1. Go to `http://localhost:3000/email`
2. Add recipients to the first text input. If you want to send to multiple emails, split them using a comma **with no space after the comma**. (e.g. "kellypham@uwblueprint.org, jasonling@uwblueprint.org")
3. Add the email body to the textbox
4. Click send 
5. Check your email and you should have received an email from kellypham@uwblueprint.org



## What should reviewers focus on?

- functionality

## Checklist
